guides/physical_device_connection.rst

```diff
@@ -0,0 +1,329 @@
+Physically Connecting to your Device
+====================================
+
+The most difficult part of setting up a new ESPHome device is the initial
+installation, which requires connecting your ESP device to a computer using a
+cable.
+
+**You only need to do this once per device.** Once you've flashed ESPHome on a
+device, you can use :doc:`the OTA updater </components/ota>` to upload new
+versions or configuration changes wirelessly.
+
+ESPHome runs on a wide variety of devices, so it's hard to list any specific
+set of tools that you need or to give instructions on how to connect. This
+guide tries to cover some of the more common flashing situations. If your
+device doesn't fit any of these situations, try and find a guide for your
+specific device in the :ref:`devices guides <devices>` or elsewhere else on the
+internet.
+
+Connecting to the ESP
+---------------------
+
+There's a wide variety of situations you might find yourself in, each of which
+requires you do something different to connect your computer to the ESP in
+order to flash it.
+
+You only need to physically connect to it once. Once you've flashed your device
+and connected it to your WiFi, you can use the `OTA (over-the-air) update
+component </components/ota.html>`_ to install software remotely.
```

Comment:
I think we can leave this out, this is already mentioned three paragraphs back.

Reply:
IMO this is a key selling  point of ESPHome, and repeating things is sometimes nice, especially since people will often skip over sections.

I can remove it if you feel strongly about it, but this wasn't repeated by accident.

Reply:
I don't feel too strongly about it, so let's leave it in then :)